### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by Mexxna** [ascott18/TellMeWhen#1898]
**What version of TellMeWhen are you using? ** 9.0.7 BC from WoW curse. 

**What steps will reproduce the problem?**
1.
2.
3.

**What do you expect to happen? What happens instead?**
TMW doesn't show cd totems (rank >= 2). At this moment im checking Searing Totem,Stoneskin and stoneclaw. Rank 1 working well and addon show it.Flame shock and Earth shock also working fine ( rank 2 and higher). I have checked name to track and name to show( changed on exactly rank 2). After agreed nothing happens. 
I tried find "back doors":
-field ( name to track I left blank) ,field what show- exactly same rank which im using. After this TMW show icon and cd but : 
--FIre totems( searing and fire nova)
--Earth Totems ( stoneclaw and stoneskin)
At this moment I have been learning rank 3. Same situation as above.
I tried write NOT totem name but ID- same situation like above.

**Screenshots and Export Strings**

**Additional Info**

**Comment by ascott18** [comment]
Please do not ignore the issue template that is provided when you create a new issue. The questions and sections within it provide me with essential information to be able to diagnose and resolve your issue. If you do not include this information, there is little I can do to help you.

If filtering totems by name, you need to enter the name of the totem. Totems in game versions that have spell ranks have the spell rank in the name of the totem. For example, `Searing Totem III`.

**Comment by Mexxna** [comment]
It was my first post on Reddit so next time I dont forget to fill
everything what u need to resolve my problem. I have matched rank totems in
TMW to rank totems in game. I have checked also spell name in both position
( addon and game)and were same. Also I wrote ID spells. But it doesnt work.
Before I report Bug? To u i have checked All problem with addon which I
could resolve.  But I couldn’t. I will send u TMW code with my settings on
Sunday- im now on vacation.

W dniu wt., 1.06.2021 o 17:30 Andrew Scott ***@***.***>
napisał(a):
